corehq/apps/hqmedia/views.py

```diff
@@ -289,6 +293,69 @@ def update_multimedia_paths(request, domain, app_id):
     })
 
 
+@method_decorator(toggles.BULK_UPDATE_MULTIMEDIA_PATHS.required_decorator(), name='dispatch')
+@method_decorator(require_can_edit_apps, name='dispatch')
+class MultimediaTranslationsCoverageView(BaseMultimediaTemplateView):
+    urlname = "multimedia_translations_coverage"
+    template_name = "hqmedia/translations_coverage.html"
+    page_title = ugettext_noop("Translations Coverage")
+
+    @use_select2_v4
+    def dispatch(self, request, *args, **kwargs):
+        return super(MultimediaTranslationsCoverageView, self).dispatch(request, *args, **kwargs)
+
+    @property
+    def parent_pages(self):
+        return [{
+            'title': _("Multimedia Reference Checker"),
+            'url': reverse(MultimediaReferencesView.urlname, args=[self.domain, self.app.get_id]),
+        }]
+
+    @property
+    def page_context(self):
+        context = super(MultimediaTranslationsCoverageView, self).page_context
+        context.update({
+            "media_types": {t: CommCareMultimedia.get_doc_class(t).get_nice_name()
+                            for t in CommCareMultimedia.get_doc_types()},
+            "selected_langs": self.request.POST.getlist('langs', []),
+            "selected_media_types": self.request.POST.getlist('media_types', ['CommCareAudio', 'CommCareVideo']),
```

Comment:
`CommCareImage` not included here? Can also use `CommCareMultimedia.get_doc_types`?

Reply:
My understanding is images typically don't get "translated", just audio and video.

Reply:
haha, that's right :)